contents/common/configuring_the_cli.tw2

```diff
@@ -13,7 +13,13 @@ After typing in the command, hit the `Enter` key.
 
 After hitting the `Enter` key, you should see a notification from the CLI that a configuration file has been written.
 
-### Important: The token above should be treated like a password and not be shared with anyone!
+### Important: The token above should be treated like a password and not be shared with anyone.
+
+   For this reason, you can delete it from your shell history, if you are using Bash, by running:
+   ```bash
+   for X in $(history | grep "exercism configure --token=" | grep -v grep | cut -d ' ' -f1) do; history -d $X; done
+   ```
+
```

Comment:
Good information to know, but not relevant here.  The problem being that this is often shared in a public support area, and removing it from their local history is not going to prevent it from being exposed.

Also, the period is surely not appropriate.  We mean the exclamation mark to echo the "Important" wording a the beginning.  It is not a passive statement, it is an emphatic statement.

Also, if we are going to mention Bash, we then might be expected to educate the user on all shells and other operating systems ideas of "shell" environments.  Their local security, while important, is not our responsibility.

I appreciate the time taken, but my vote is "no" on 95.  It isn't bad, it just doesn't reach far enough, and reaches too far for the scope of what we should cover.

Reply:
The thing here is that you are telling people to treat the token as a password, and passwords should never be kept plain in the history. I think your configure command you prompt user for the password then.
those are my 2 cents

Reply:
Anyway, `exercism configure` shows the token in plain text. So it is pretty much the same as running `history`

Reply:
Yes, but for sharing the problem you are having, you should use `exercism debug` which obfuscates this.  And the history of the terminal can be avoided in some cases by prepending a space.

My point it, local history exposure of sensitive things is a different subject, sharing the "password" or token in a public forum is to be avoided.

Reply:
I apologize for the friction.  I do agree with the sentiment.  I just think that local system security is hinted at strong enough with the exclamation mark and the first word of that sentence, to raise awareness to not share it.  And if on a shared system (or a system that sits in front of a camera, or that is being screenshot by management) where someone else can gain this information, it is not something that we can really address in this context.

Reply:
/cc @iHiD